features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunInterface.h

```diff
@@ -23,18 +23,34 @@
  * \brief Struct ws_rpl_info Wi-SUN router RPL information.
  */
 typedef struct ws_rpl_info {
-    /** Address prefix given to devices in network  set to 0 if not available*/
-    uint8_t ipv6_prefix[8];
-    /** IID of router */
-    uint8_t ipv6_iid[8];
+    /** IPv6 Global Address of Router Node*/
+    uint8_t global_addr[16];
+    /** IPv6 Link Local Address of Router Node*/
+    uint8_t link_local_addr[16];
     /** Router dodag id */
     uint8_t rpl_dodag_id[16];
     /** Router instance identifier */
     uint8_t instance_id;
     /** RPL version number */
     uint8_t version;
+    /** RPL DODAG node current Rank */
+    uint16_t curent_rank;
+    /** RPL Primary Parent Rank */
+    uint16_t primary_parent_rank;
+    /** RPL Primary Parent Address */
+    uint8_t rpl_parent_addr[16];
 } ws_rpl_info_t;
 
+/**
+ * \brief Struct ws_radio_info Wi-SUN router Radio Quality information.
+ */
```

Comment:
I would suggest larger Scope API A bit similar than in the nano stack 
parent link local address, RSL in RSL out, join state,PAN_ID You get these from the same interface and would match other than RPL information

Reply:
perhaps make 
ws_stack_state
Global Address,
Link Local address,
Parent Link Local address
RSL_in, RSL_out
Join state
PAN_ID

Reply:
Done.